main/secrets.js

```diff
@@ -8,8 +8,9 @@ const secrets = {};
  * @param {Secret} secret 
  */
 module.exports.save = function save(secret) {
-    const key = secret.getLookupKey();
-
+    //const key = secret.getLookupKey();
```

Comment:
Is this no longer necessary?

As far as I recall, this is in place because the management part (the UI) is putting the `key` in the query string, and we therefore need the key to be query string safe.

Have you tested if the UI still works?